patroni/dcs/__init__.py

```diff
@@ -1557,42 +1571,45 @@ def __get_patroni_cluster(self, path: Optional[str] = None) -> Cluster:
         """
         if path is None:
             path = self.client_path('')
-        cluster = self._load_cluster(path, self._cluster_loader)
+        cluster = self._load_cluster(path, self._patroni_cluster_loader)
         if TYPE_CHECKING:  # pragma: no cover
             assert isinstance(cluster, Cluster)
         return cluster
 
-    def is_citus_coordinator(self) -> bool:
-        """:class:`Cluster` instance has a Citus Coordinator group ID.
+    def is_coordinator_cluster(self) -> bool:
```

Comment:
Here we check our own configuration.
```suggestion
    def is_formation_coordinator(self) -> bool:
```